Doc/library/datetime.rst

```diff
@@ -1347,22 +1347,22 @@ Instance methods:
 
    where ``yday = d.toordinal() - date(d.year, 1, 1).toordinal() + 1``
    is the day number within the current year starting with ``1`` for January
-   1st. The :attr:`tm_isdst` flag of the result is set according to the
+   1st. The :attr:`tm_isdst<time.struct_time.tm_isdst>` flag of the result is set according to the
    :meth:`dst` method: :attr:`.tzinfo` is ``None`` or :meth:`dst` returns
-   ``None``, :attr:`tm_isdst` is set to ``-1``; else if :meth:`dst` returns a
-   non-zero value, :attr:`tm_isdst` is set to ``1``; else :attr:`tm_isdst` is
+   ``None``, :attr:`tm_isdst<time.struct_time.tm_isdst>` is set to ``-1``; else if :meth:`dst` returns a
```

Comment:
Too mach references to the same this in close proximity is not good. It is enough to keep the first link and silence others.
```suggestion
   ``None``, :attr:`!tm_isdst` is set to ``-1``; else if :meth:`dst` returns a
```